api/graphql/schema.graphql

```diff
@@ -1793,12 +1796,15 @@ type Mutation {
   # Application mutations
   archiveApplication(id: ID!): PoolCandidate
     @guard
-    @can(ability: "archiveApplication", find: "id")
+    @can(ability: "archive", find: "id")
   createApplication(userId: ID!, poolId: ID!): PoolCandidate
+    @can(ability: "createDraft")
   deleteApplication(id: ID!): Boolean
     @guard
-    @can(ability: "deleteApplication", find: "id", model: "PoolCandidate")
+    @can(ability: "delete", find: "id", model: "PoolCandidate")
   submitApplication(id: ID!, signature: String!): PoolCandidate
+    @guard
+    @can(ability: "submit", find: "id")
```

Comment:
If we're not checking application Application details in these policy methods, then lets add the validation.

Reply:
All relevant validation exists within [`api/app/GraphQL/Validators/Mutation`](https://github.com/GCTC-NTGC/gc-digital-talent/tree/feature_5793_update-pool-candidate-policy/api/app/GraphQL/Validators/Mutation)

Reply:
Thanks! I always forget these mutations have their own handlers, not just Lighthouse magic.